### PR TITLE
Add Django Documentation

### PR DESCRIPTION
**Pull request by Moghedrin**
Add documentation for the new, official Django images.

### CODE REVIEW

File: django/content.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+having to add a `Dockerfile` to your project.
+
+    docker run --name some-django-app -v "$(pwd)":/usr/src/app -w /usr/src/app -p 8000:8000 -d django bash -c "pip install -r requirements.txt && python manage.py runserver 0.0.0.0:8000"
+
+## Bootstrap a new Django Application
+
+If you want to generate the scaffolding for a new Django project, you can do the
+following:
+
+    docker run -it --rm --user "$(id -u)" -v "$(pwd)":/usr/src/app -w /usr/src/app django django-admin.py startproject mysite
```
Comment (tianon) [inline django/content.md]: :wink:

**Comment by Moghedrin** [comment]
ping @tianon

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
LGTM

**Pull request merged**